bentoml/_internal/configuration/containers.py

```diff
@@ -535,5 +541,12 @@ def duration_buckets(
             )
         return DEFAULT_BUCKET
 
+    @providers.SingletonFactory
+    @staticmethod
+    def logging_formats(
```

Comment:
I prefer to make this `logging_formatting` to be consistent with the config level.

Reply:
I could change it, but would there be a use case for someone to have one trace id format for the api server and a different format for the runner?

Reply:
Oh I was saying just to change the naming to make it consistent